ruby/lib/components/shell/oh_my_zsh.rb

```
@@ -42,12 +45,46 @@ def install!
+        raise ".zshrc file not found"
+      end
+
+      zshrc_content = File.read(ZSHRC)
+      
+      plugins_string = "plugins=("
+      PLUGINS.each do |plugin|
+        plugins_string += " #{plugin}"
+      end
+      plugins_string += ")"
+
+      if zshrc_content.gsub!(/^[^#]*plugins=\([^)]*\)/, "#{plugins_string}")
```

RuboCop (1.81.7)

[convention] Prefer `to_s` over string interpolation.

(Style/RedundantInterpolation)